File: server/src/main/java/cn/iinti/atom/service/base/alert/events/SensitiveOperationEvent.java
package cn.iinti.atom.service.base.alert.events;


public record SensitiveOperationEvent(String user,
                                      String api,
                                      String params) {
    public String getMessage() {
        return """
                \
                user: %s
                api: %s
                params: %s""".formatted(user, api, params);
    }
}

File: server/src/main/kotlin/cn/iinti/atom/service/base/alert/events/SensitiveOperationEvent.kt
package cn.iinti.atom.service.base.alert.events

data class SensitiveOperationEvent(
    val user: String,
    val api: String,
    val params: String
) {
    fun getMessage(): String {
        return """
                user: $user
                api: $api
                params: $params
                """.trimIndent()
    }
}